Frontend/ui-customer/src/Flow.purs

```diff
@@ -4041,3 +4043,27 @@ checkForSpecialZoneAndHotSpots state (ServiceabilityRes serviceabilityResp) lat
       liftFlowBT $ runEffectFn1 locateOnMap locateOnMapConfig { points = points, zoomLevel = zoomLevel, labelId = getNewIDWithTag "LocateOnMapPin"}
     else pure unit
   else pure unit
+
+applyReferralCode :: String -> FlowBT String ST.ReferralStatus
```

Comment:
can we keep the code away from flow file, which is not actually related to Flow and rather are helper functions